src/rpc/rpcHandler.ts

```diff
@@ -542,8 +541,10 @@ export class RpcHandler implements IRpcEndpoint {
                 entryPoint,
                 queuedUserOperations,
                 false,
-                deepHexlify(stateOverrides)
+                deepHexlify({ ...stateOverrides })
             )
+        } catch (err) {
+            this.logger.error({ error: err }, "Second simulations fail")
```

Comment:
```suggestion
            this.logger.error({ err }, "Second simulations fail")
```

we should use `err` for logging an Error: https://getpino.io/#/docs/api?id=mergingobject-object

> If the object is of type Error, it is wrapped in an object containing a property err ({ err: mergingObject }). This allows for a unified error handling flow.

or just `this.logger.error(err, "Second simulations fail")`